frontend/components/EventPage/DateInterval.tsx

```diff
@@ -5,13 +5,18 @@ import styled from 'styled-components'
 
 import { LIGHT_GRAY } from '../../constants'
 
+
 const dateIntervalString = (startDate: Date, endDate: Date): string => {
   const [start, end] = [startDate, endDate].map((d) => moment(d))
+    
   if (![start, end].every((d) => d.isValid())) {
     return 'Invalid date range'
   }
-  const [startDateString, endDateString] = [start, end].map((d) =>
-    d.format('dddd, MMM D'),
+  const [startDateString, endDateString] = [start, end].map((d) => {
+    const dateDifference = start.startOf("day").diff(moment().startOf("day"), "days");
+    if (dateDifference == 0) return "Today";
+    if (dateDifference == 1) return "Tomorrow";
+    return d.format('dddd, MMM D')}
```

Comment:
It looks like you're encountering a few typescript errors from the style checker (see details [here](https://app.circleci.com/pipelines/github/pennlabs/penn-clubs/1887/workflows/e9dfd11f-da9c-4fe0-8feb-4e70db8e6082/jobs/4831)).

To fix this, you can run `yarn clean` to try to automatically fix some issues and `yarn lint` to see style errors. If you use VSCode, you can use the code workspace provided, which will format your files on save.